governing-documents/inactivity.md

```diff
@@ -9,24 +9,24 @@ nav_order: 8
 
 # Inactivity Policies
 
-As part of the normal lifecycle of a project codebases and maintainers come and go. This document formalizes a Hyperledger wide policy for moving maintainers to emeritus status when a project does not have an explicit policy of their own, and the responsibilities of the Technical Advisory Council (TAC), Hyperledger Foundation staff, and maintainers in this process.
+As part of the normal lifecycle of a project, codebases and maintainers come and go. This document outlines a LF Decentralized Trust-wide policy for moving maintainers to emeritus status when a project's TSC does not have an explicit policy of its own, as well as the responsibilities of the Technical Advisory Council (TAC), LF Decentralized Trust staff, and maintainers in this process.
```

Comment:
Should the organizational policy be parallel with the project’s TSC policy, or only in effect when the TSC doesn’t have a policy?  I think the latter might be too limiting.

Reply:
I'd vote for it providing the defaults in case the TSC doesn't feel the need to define their own. 
On Cacti we always try to go with the defaults so it would be great for us to preserve maintainer (TSC member) resources/time.

For those who would like to have more custom policies, this would still give them the option to define things for themselves.

It should be mentioned that some things from the organizational policy should be non-overridable, DCO being the example coming to mind.

Reply:
@swcurran are you considering a case where the project may have lenient policy as opposed to being stricter?

Reply:
I'm thinking of the case where the project is not enforcing their policy.  I think even if they have a policy, this policy must be used.  For example, when the inactivity in the Project's TSC itself.